src/NLopt.jl

```diff
@@ -454,28 +477,39 @@ srand_time() = nlopt_srand_time()
 # Objective function:
 
 const _EMPTY_VECTOR = Cdouble[]
+const _EMPTY_MATRIX = zeros(Cdouble, 0, 0)
 
 @inline function _get_empty_vector()
     @assert isempty(_EMPTY_VECTOR) "Builtin empty vector modified by user"
     return _EMPTY_VECTOR
 end
 
+@inline function _wrap_pointer(o::Opt, p, sz, field, isgrad)
```

Comment:
```suggestion
@inline function _wrap_pointer(
    o::Opt,
    p::Ptr,
    sz::Tuple,
    field::Symbol,
    isgrad::Bool,
)
```

Thoughts on typing these arguments for documentation sake? This method is a little scary to skim otherwise, even if it is fairly clear on a close reading.